content/sensu-enterprise/3.1/integrations/ec2.md

```diff
@@ -102,14 +108,33 @@ type           | Array
 allowed values | `ok`, `warning`, `critical`, `unknown`
 example        | {{< highlight shell >}} "severities": ["critical", "unknown"]{{< /highlight >}}
 
-timeout      | 
+accounts     | 
 -------------|------
-description  | The handler execution duration timeout in seconds (hard stop).
+description  | List of trusted client names and roles? See [`name` and `role_arn` attributes][7].
 required     | false
-type         | Integer
-default      | `10`
-example      | {{< highlight shell >}}"timeout": 30{{< /highlight >}}
+type         | Array of hashes
+example      | {{< highlight shell >}}"accounts": [
+  {
+    "name": "sensuapp",
+    "role_arn": "arn:aws:iam::921689585014:role/CrossAccountSignin"
+  }
+]{{< /highlight >}}
+
+#### `accounts` attributes
 
+name         | 
+-------------|------
+description  | AWS account name?
```

Comment:
yes, this is expected to be a name for the AWS account. This account name isn't significant outside Sensu configuration. The integration uses the value of the `account` attribute in the `client.ec2` attributes as the identifier for selecting the corresponding ec2 account in these settings.